pkg/sql/plan/build_show.go

```diff
@@ -110,10 +115,23 @@ func buildShowCreateTable(stmt *tree.ShowCreateTable, ctx CompilerContext) (*Pla
 		}
 		typ := types.Type{Oid: types.T(col.Typ.Id)}
 		typeStr := typ.String()
+		if types.IsDecimal(typ.Oid) { //after decimal fix,remove this
+			typeStr = "DECIMAL"
+		}
 		if typ.Oid == types.T_varchar || typ.Oid == types.T_char {
 			typeStr += fmt.Sprintf("(%d)", col.Typ.Width)
 		}
-		createStr += fmt.Sprintf("`%s` %s %s%s", colName, typeStr, nullOrNot, hasAttrComment)
+		updateOpt := ""
+		if col.OnUpdate != nil {
+			if f, ok := col.OnUpdate.Expr.(*plan.Expr_F); ok { //? other type
+				fStr, err := convertFunc2Str(f.F)
+				if err != nil {
+					return nil, err
+				}
+				updateOpt = " ON UPDATE " + fStr
+			}
+		}
+		createStr += fmt.Sprintf("`%s` %s %s%s%s", colName, typeStr, nullOrNot, updateOpt, hasAttrComment)
```

Comment:
to recover `on update` attribute on column.
According to yuanning's suggestion， I will add `originString` in OnUpdate filed to avoid this deserialization

Reply:
My understanding of the following queries is sufficient for you:
```
select * from mo_tables.rel_create where ...
```

Reply:
emmm, here I use `show create table` clause to get the ddl.
I tried `select * from mo_tables.rel_create` but got err: `table not exists`

Reply:
mo_catalog.mo_tables